app/views/items/index.html.erb

```diff
@@ -2,9 +2,9 @@
 
 <div id="items">
   <div class="container">
-      <div class="row ">
-        <% @items.each do |item| %>
-          <div class="col-sm-3 mb-4">
+    <div class="row ">
+      <% @items.each do |item| %>
+        <div class="col-sm-3 mb-4">
           <div class="card">
             <img class="card-img-top" src="https://via.placeholder.com/400x300" alt="Card image cap">
```

Comment:
What does this placeholder do?

Reply:
Not our business. Have a look at the path. This file is listed as I did a code cleanup.